### PR TITLE
Branch add calorie (Can merge)

### PR DESCRIPTION
**Pull request by jundatan**

### CODE REVIEW

File: src/main/java/seedu/address/model/client/Calorie.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Calories should only contain numbers, and it should be at least 4 digits long";
+    public static final String VALIDATION_REGEX = "\\d{4,}";
+    public final String value;
+
+    /**
+     * Constructs a {@code Calorie}.
+     *
+     * @param calorie A valid phone number.
```
Comment (dfordarius) [inline src/main/java/seedu/address/model/client/Calorie.java:20]: Need amendment for @param line

File: src/main/java/seedu/address/model/client/Calorie.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+     */
+    public Calorie(String calorie) {
+        requireNonNull(calorie);
+        checkArgument(isValidCalorie(calorie), MESSAGE_CONSTRAINTS);
+        value = calorie;
+    }
+
+    /**
+     * Returns true if a given string is a valid phone number.
```
Comment (dfordarius) [inline src/main/java/seedu/address/model/client/Calorie.java:29]: Need amendment

File: src/main/java/seedu/address/model/client/Calorie.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.client;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's calorie intake in the fitbook.
```
Comment (dfordarius) [inline src/main/java/seedu/address/model/client/Calorie.java:7]: Would be good to standardize as 'FitBook'

Comment (dfordarius) [reply src/main/java/seedu/address/model/client/Calorie.java:7]: and Client's

File: src/test/java/seedu/address/model/client/NameContainsKeywordsPredicateTest.java [hunk 4]
```diff
@@ -68,8 +68,9 @@ public void test_nameDoesNotContainKeywords_returnsFalse() {
         assertFalse(predicate.test(new ClientBuilder().withName("Alice Bob").build()));
 
         // Keywords match phone, email and address, but does not match name
```
Comment (dfordarius) [inline src/test/java/seedu/address/model/client/NameContainsKeywordsPredicateTest.java:70]: update to include calories

**Review by OliviaJHL — APPROVED** [review]
It is good to me

**Pull request merged**